org.civicrm.bootstrapcivicrm/scss/civicrm/vacancies/_vacancies.scss

```diff
@@ -45,6 +70,37 @@
         }
       }
     }
+
+    @media (max-width: 1340px) {
```

Comment:
Are you sure those media-queries will work?
I know they should, but since the markup lacks the proper viewport setup, I've ended up using `max-device-width` instead.
Please, try those in chrome to be sure.